packages/utilities/psammead-assets/src/svgs.stories.jsx

```diff
@@ -56,50 +56,17 @@ getSVG.propTypes = {
   height: number,
 };
 
-storiesOf('SVGs', module)
-  .addDecorator(withKnobs)
-  .add(
-    'news',
-    () => {
-      const height = numberKnob('Height', 24);
-      const NewsSvg = getSVG({ ...news, height });
-      return NewsSvg;
-    },
-    { notes },
-  )
-  .add(
-    'Igbo',
-    () => {
-      const height = numberKnob('Height', 24);
-      const IgboSvg = getSVG({ ...igbo, height });
-      return IgboSvg;
-    },
-    { notes },
-  )
-  .add(
-    'Yoruba',
-    () => {
-      const height = numberKnob('Height', 24);
-      const YorubaSvg = getSVG({ ...yoruba, height });
-      return YorubaSvg;
-    },
-    { notes },
-  )
-  .add(
-    'Thai',
-    () => {
-      const height = numberKnob('Height', 24);
-      const ThaiSvg = getSVG({ ...thai, height });
-      return ThaiSvg;
-    },
-    { notes },
-  )
-  .add(
-    'Pidgin',
-    () => {
-      const height = numberKnob('Height', 24);
-      const PidginSvg = getSVG({ ...pidgin, height });
-      return PidginSvg;
-    },
-    { notes },
-  );
+const stories = storiesOf('SVGs', module).addDecorator(withKnobs);
+
+Object.keys(svgs)
+  .filter(key => key !== 'BBC_BLOCKS')
```

Comment:
this might be just me being picky, but can we get a more descriptive name here than just `key`? Something like `service` or `serviceName`, etc.

Reply:
Hmm ive changed it to svgName, but im not sure now :P
Its not really service or service name as `BBC_BLOCKS` isnt a service

Reply:
I think `svgName` already makes it more readable. `key` wasn't very clear imo

Reply:
shouldn't this also be updated on the line below and in the test file?

Reply:
done :)